autogen/agentchat/groupchat.py

```diff
@@ -42,6 +42,15 @@ class GroupChat:
         - "manual": the next speaker is selected manually by user input.
         - "random": the next speaker is selected randomly.
         - "round_robin": the next speaker is selected in a round robin fashion, i.e., iterating in the same order as provided in `agents`.
+        - a customized speaker selection function: if passed in a function, it will be called with the following parameters:
```

Comment:
```suggestion
        - a customized speaker selection function (Callable): if passed in a function, it will be called with the following parameters:
```